cci_cameroon/analysis/eda_ifrc/eda_ifrc.py

```diff
@@ -0,0 +1,572 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.13.2
+#   kernelspec:
+#     display_name: cci_cameroon
+#     language: python
+#     name: cci_cameroon
+# ---
+
```

Comment:
Generally looks good.  You may consider moving the images to the outputs folder.